File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/C++/4_Sorting.cpp
#include <iostream>
#include <algorithm> //de sd ham sorting san trong thu vien
#include <vector>

using namespace std;

int main(){
    int arr[] = {3,2,1};

    //sorting array
    int size = sizeof(arr) / sizeof(int);
    sort(arr, arr + size); //ham sort truyen address head & tail
    cout << "arr: ";
    for (int i=0; i<size; i++){
        cout << arr[i] << " ";
    }
    cout << endl;

    //sorting vector
    vector<int> list(arr, arr+size);
    sort(list.begin(), list.end());
    cout << "list: ";
    for (int i=0; i<list.size(); i++){
        cout << list[i]<< " ";
    }
}

File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/Java/4_Sorting.js
import java.util.ArrayList;
import java.util.Arrays;

public class SortingInJava {
    public static void main(String[] args) {
    //Java sorting array
		int[] arr = {3,2,1};
    Arrays.sort(arr);
    System.out.print("arr: ");
    for (int i=0; i<arr.length; i++){
        System.out.print(arr[i]+" ");
    }
    System.out.println();
    }
}

//---------------------------------------------
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

public class SortingInJava {
    public static void main(String[] args) {
    //Java sorting list
		System.out.print("list: ");
    List<Integer> list = new ArrayList<>(List.of(3,2,1));
    Collections.sort(list);
    for (int i=0; i<list.size(); i++){
        System.out.print(list.get(i)+" ");
    }
    }
}